From 515e1f76cbcd0ed7c0ebf3c58fbf2ab977592343 Mon Sep 17 00:00:00 2001
From: Wang Jie <dave.wang@rock-chips.com>
Date: Mon, 14 Oct 2019 13:56:29 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: set
 CONFIG_TOUCHSCREEN_GSLX6801=y.

The touch screen driver is for the new rk3368 evb board.

Change-Id: I213a461c956a219ae1969ac30dc23e587d3d723e
Signed-off-by: Wang Jie <dave.wang@rock-chips.com>
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 636828d495b5..4f3976ddf22c 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -431,6 +431,7 @@ CONFIG_TABLET_USB_GTCO=y
 CONFIG_TABLET_USB_HANWANG=y
 CONFIG_TABLET_USB_KBTAB=y
 CONFIG_INPUT_TOUCHSCREEN=y
+CONFIG_TOUCHSCREEN_GSLX6801=y
 CONFIG_TOUCHSCREEN_GSLX680_PAD=y
 CONFIG_TOUCHSCREEN_GSLX680_VR=y
 CONFIG_TOUCHSCREEN_GSL3673=y
-- 
2.35.3

